app/models/concerns/serializable.rb

```diff
@@ -46,7 +46,9 @@ def xml_schema
 
       def displayed_type(field, type)
         # Get a string of options for this field, if Incident::OPTIONS_PER_FIELD[field] is defined.
-        options = Incident::OPTIONS_PER_FIELD[field].try { |opts| opts.map { |o| "'#{o}'" }.join(', ') }
+        options = Incident::OPTIONS_PER_FIELD[field].try do |opts|
+          opts.map { |o| "'#{o}'" }.join(', ').sub("&", "&amp;")
```

Comment:
Hm, so I'm actually going to take this out of this PR because it was addressed in a previous PR. I think you're right and maybe we should make a low-priority issue to do that.